python/ingestion/url_file_to_gcs.py

```diff
@@ -64,10 +76,23 @@ def download_first_url_to_gcs(url_list, gcs_bucket, dest_filename,
             dest_filename)
         return
 
-    # Download URL locally, upload to bucket and remove local file
-    local_path = local_file_path(dest_filename)
-    with file_from_url, open(local_path, 'wb') as f:
+    # Download the contents of the URL to a local file
+    url_local_path = local_file_path(dest_filename)
+    # Downloads the current file in GCS to a local file
+    gcs_local_path = local_file_path("gcs_local_file")
```

Comment:
Do we have an idea of what memory limitations we have and whether any of the existing data sets will run afoul of those?

Reply:
I haven't tested this out on all our data sources but so far memory hasn't been a problem. I believe the max memory allocation is 4GB for cloud run services. If this isn't large enough we may have to revisit your idea of using hashes (I was unable to get that working). It wasn't very straightforward to use the same hashing algorithm used by GCS.

Reply:
sg, I don't think anything we have will exceed that